samples/sample_multi_transcode/include/pipeline_transcode.h

```diff
@@ -88,7 +85,7 @@ const mfxF64  MCTF_LOSSLESS_BPP = 0.0;
 
 namespace TranscodingSample
 {
-    extern mfxU32 MFX_STDCALL TranscodeRoutine(void   *pObj);
+    extern void MFX_STDCALL TranscodeRoutine(void   *pObj);
```

Comment:
1. Do we really need this extern declaration here?
2. I don't think we need MFX_STDCALL anymore.

Reply:
Agree. extern and MFX_STDCALL are useless here